beginner_source/saving_loading_models.py

```diff
@@ -13,14 +13,14 @@
 
 1) `torch.save <https://pytorch.org/docs/stable/torch.html?highlight=save#torch.save>`__:
    직렬화된 객체를 디스크에 저장합니다. 이 함수는 Python의
-   `pickle <https://docs.python.org/3/library/pickle.html>`__ 을 사용하여 직렬화합니다
+   `pickle <https://docs.python.org/3/library/pickle.html>`__ 을 사용하여 직렬화합니다.
    이 함수를 사용하여 모든 종류의 객체의 모델, Tensor 및 사전을 저장할 수 있습니다.
 
 2) `torch.load <https://pytorch.org/docs/stable/torch.html?highlight=torch%20load#torch.load>`__:
    `pickle <https://docs.python.org/3/library/pickle.html>`__\ 을 사용하여
-   저장된 객체 파일들을 역직렬화하여 메모리에 올립니다. 이 함수는 데이터를 장치에
-   불러올 때도 사용합니다.
-   (`장치간 모델 저장하기 & 불러오기 <#device>`__ 참고)
+   저장된 객체 파일들을 역직렬화하여 메모리에 올립니다. 이 함수는 데이터를 장치에 불러
+   때도 사용합니다.올
```

Comment:
글자가 잘못 들어간 것 같습니다. :)

```suggestion
   올 때도 사용합니다
```